schema.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE animals (
     escape_attempts INT,
     neutered BOOLEAN,
     weight_kg DECIMAL,
+    ALTER TABLE animals ADD species varchar(100),  /*ALTER command is for changing the structure(It help us to adding a forgotten column, for our case it's species column)*/
```

Comment:
- [ ] Please kindly take this command out of your `CREATE TABLE` statement.  

## Look at the syntax:

![image](https://user-images.githubusercontent.com/57421823/151321842-952d577c-18f4-466e-a4f1-f26154185fba.png)

## Result:

![image](https://user-images.githubusercontent.com/57421823/151321964-3501b546-35cb-43a9-8102-b1dee0441019.png)

That way, the alteration of the table comes after its original creation, and the change will be safer 💪 

Looking to your resubmission so it gets approved fast with this small change! 💪 💪 💪